### PR TITLE
Added boolean for logger

### PR DESCRIPTION
**Pull request by need4swede**
### Example

Enable:
`ff = FFmpeg()`

Disable:
`ff = FFmpeg(enable_log=False)`

Quick and easy way to enable 'silent' executions
These changes are not OS dependent, but let me know if should be creating the PR to a different branch

### CODE REVIEW

**Comment by amoh-godwin** [comment]
@need4swede thank you for this PR. I like the idea. You are targeting the right branch for this PR. But certain codes have been deleted which I do not know why.

File: pyffmpeg/__init__.py [hunk 1]
```diff
@@ -225,21 +243,15 @@ def options(self, opts):
...
         out = Popen(options, shell=SHELL, stdin=PIPE, stdout=PIPE, stderr=PIPE)
         self._ffmpeg_instances['options'] = out
-        stderr = str(out.stderr.read(), 'utf-8')
-        if stderr and 'Output #0' not in stderr:
-            self.error = stderr.rsplit('\r\n', maxsplit=2)[-2]
-            self.logger.error(self.error)
-            raise Exception(self.error)
-        else:
-            self.error = ''
-            self.logger.info('Conversion Done')
```
Comment (amoh-godwin) [inline pyffmpeg/__init__.py:242]: Standard error, in this case, `stderr`, refers to whatever text was printed to the console whiles the application was still running. It is assumed to have been an error message. Standard output is the text that is printed after an application exits, it is the return statement of the application. But now, from what I see, your code is just putting every standard error text into the error variable. This code checks if there was an "output" information, which is the information FFMpeg prints on the output file, if there wasn't then was an error.

**Review by amoh-godwin — CHANGES_REQUESTED** [review]
I actually like the idea. If the codes you deleted should be rewritten, rewrite them I will be checking back very soon to merge the PR. Thank you!

**Comment by need4swede** [comment]
Odd, I don't remember deleting anything...let me look over my files. Perhaps my version didn't have them? I installed it via pip. I'll look things over and will get back to you

**Comment by need4swede** [comment]
Updated with lines

**Pull request merged**